extend.php

```diff
@@ -50,7 +50,8 @@
         ->attributes(TrelloAttributes::class),
 
     (new Extend\Settings())
-        ->serializeToForum('trelloDefaultBoardId', 'blomstra-trello.default_board_id', 'strval'),
+        ->serializeToForum('trelloDefaultBoardId', 'blomstra-trello.default_board_id', 'strval')
+        ->serializeToForum('trelloLastUsedLaneId', 'blomstra-trello.last_used_lane_id', 'strval'),
```

Comment:
I'm actually thinking we should hide all of this from the api, unless the user has Trello permission. 

Can you move these to https://github.com/blomstra/flarum-ext-trello/blob/master/src/TrelloAttributes.php please and remove the `Extend\Settings`